src/main/python/main.py

```diff
@@ -37,20 +44,21 @@ def run(self):  # 2. Implement run()
             url = "https://github.com/" + self.REPO + "/releases/tag/" + tag
             utils.display_blocking_popup(
                "<p>A new version of the AYAB desktop software has been released! You can download version <strong>" + tag + "</strong> using this link:<br/><br/><a href='" + url + "'>" + url + "</a></p>")
-        return self.app.exec()  # 3. End run() with this line
+        return cast(int,self.app.exec())  # 3. End run() with this line
```

Comment:
mypy doesn't get the whole decorator thing so it doesn't know `app` is a `ApplicationContext` not a `Func`, so we help it out here.